packages/components/src/List/ListComposition/SortBy/SortBy.component.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Navbar, NavDropdown, Nav, NavItem, MenuItem } from 'react-bootstrap';
+import uuid from 'uuid';
+import isUndefined from 'lodash/isUndefined';
+
+import { useListContext } from '../context';
+
+function SortBy(props) {
+	const { id, options, selected, isDescending: isDescendingProp, onChange, onOrderChange } = props;
+	const { sortParams, setSortParams, t } = useListContext();
+
+	// Sort by
+	let selectedOption;
```

Comment:
Could you make an outside function to do this else if. It's harder to follow the mutation of the selectedOption variable